_search-plugins/star-tree-index.md

```diff
@@ -7,71 +7,117 @@ nav_order: 54
-- **Green**: In a `term` query that searches for the number of aggregation requests, the `status` equals `200`. Because the query only contains a value from the `status` dimension, the query traverses the `200` node's child star node, which contains the aggregated value of all the `port` child nodes.
-- **Red**: In a `term` query that searches for the average request size aggregation, the port equals `5600`. Because the query does not contain a value from the `status` dimension, the query traverses a star node and returns the aggregated result from the `5600` child node.
+### How queries use the star-tree
 
-Support for the `Terms` query will be added in a future version. For more information, see [GitHub issue #15257](https://github.com/opensearch-project/OpenSearch/issues/15257).
-{: .note}
+The following diagram shows a star-tree index created for this example and three example query paths.
+
+<img src="{{site.url}}{{site.baseurl}}/images/star-tree-index.png" alt="A star-tree index containing two dimensions and two metrics">
+
+The colored arrows show three query examples:
+
+* **Blue arrow**: Multi-term query with metric aggregation
+  The query filters on both `status = 200` and `port = 5600` and calculates the sum of request sizes.
+
+  * OpenSearch follows the path: `Root → 200 → 5600`
+  * It retrieves the metric from Doc ID 1, where `Sum(size) = 988`
+
+* **Green arrow**: Single-term query with metric aggregation
+  The query filters on `status = 200` only and computes the average request latency.
+
+  * OpenSearch follows the path: `Root → 200 → *`
+  * It retrieves the metric from Doc ID 5, where `Avg(latency) = 70`
+
+* **Red arrow**: Single-term query with metric aggregation
+  The query filters on `port = 8443` only and calculates the sum of request sizes.
+
+  * OpenSearch follows the path: `Root → * → 8443`
+  * It retrieves the metric from Doc ID 7, where `Sum(size) = 1111`
+
+These examples show how OpenSearch selects the shortest path in the star-tree and uses pre-aggregated values to process queries efficiently.
+
+## Limitations
+
+Star-tree indexes have the following limitations:
+
+- Enable a star-tree index only for indexes whose data is not updated or deleted: star-tree indexes do not support updates or deletions. To enforce this policy, set `index.append_only.enabled` to `true` for the index you want to enable star-tree index on.
+- Use a star-tree index for aggregation queries only if the queried fields are a subset of the star-tree's dimensions and the aggregated fields are a subset of the star-tree's metrics.
+- Once enabled, a star-tree index cannot be disabled. In order to disable a star-tree index, the data in the index must be reindexed without the star-tree mapping. Changing a star-tree configuration also requires reindexing data.
```

Comment:
Regarding https://github.com/opensearch-project/documentation-website/pull/10132/commits/6848b61ec264e7512fed7c973912d768fb5ed6fd#r2162545785.

I think this is a good place to include this information. Something like "Searches using star-tree can however be disabled by this setting.

Basically setting `indices.composite_index.star_tree.enabled` to false will still create index structures related to star-tree however, the searches will be directed to default path. I'm not sure how to frame this information in a ncie way. @kolchfa-aws Please feel free to reword this section.

Reply:
Thanks! Reworded.